lib/org/chromium/webidl/IDLFileContents.js

```diff
@@ -32,5 +31,11 @@ foam.CLASS({
       required: true,
       final: true,
```

Comment:
I believe you mentioned previously that `required` and `final` should be removed due to something weird going on in the DAOs. Should I remove them from this file?